tensorflow/python/data/kernel_tests/prefetch_test.py

```diff
@@ -49,6 +52,20 @@ def testPrefetchWithSlack(self, buffer_size, slack_period):
         dataset, buffer_size, slack_period=slack_period)
     self.assertDatasetProduces(dataset, expected_output=range(100))
 
+  @test_util.run_v1_only("graph-mode specific test")
+  def testSkipEagerPrefetchCancellation(self):
+    sleep_microseconds = 1000000
+    dataset = dataset_ops.Dataset.range(10).apply(
```

Comment:
use an input pipeline that hangs instead of one that is prone to flakiness because of reliance on time

Reply:
Done